WordOfTheDay.Repository/WordsRepository.cs

```diff
@@ -18,12 +18,10 @@ public WordsRepository(WordContext context)
         }
         public async Task<WordCount> WordOfTheDay()
         {
-            var time = DateTime.Today.ToUniversalTime();
-
             var wordOfTheDayType = await _context.Words
-                .Where(word=>word.AddTime > time)
+                .LaterThan()
                 .GroupBy(word => word.Text, (text, words) => new { text, words = words.Count(word => word.Text == text) })
```

Comment:
Can you put this GroupBy to extension also. You use this code at least two times

Reply:
I understood the problem, but I already tried to fix this and put it into an extension. The main issue is that these two .GroupBy seem to be the same but they are not. The first one is (text, words) => new { text, words = words.Count(word => word.Text == text) and the second one is (text, words) => new WordCount(text, words.Count(word => word.Text == text)). The obvious solution is to rewrite the first to WordCount instead of tuple and after that write an extension. Unfortunately, in this case, I receive the following exception in the browser: 
![image](https://user-images.githubusercontent.com/87187545/148783385-705d4e88-1577-48c7-b2c1-bebb909ad1f2.png) The other solution is to rewrite the other .GroupBy into the tuple, but it causes the necessity to rewrite a lot of code to transform this query of tuples to query of WordCounts. If the second solution seems okay for you, I will do it this way.